addons/hr_presence/models/hr_employee.py

```diff
@@ -110,13 +110,21 @@ def write(self, vals):
         return super().write(vals)
 
     def action_open_leave_request(self):
-        self.ensure_one()
+        context = {
+            'default_employee_ids': self.ids,
+            'default_date_from': fields.Date.today(),
+            'default_date_to': fields.Date.today(),
+            'default_name': _('Unplanned Absence'),
+        }
+
         return {
             "type": "ir.actions.act_window",
-            "res_model": "hr.leave",
+            "res_model": "hr.leave.generate.multi.wizard",
             "views": [[False, "form"]],
             "view_mode": 'form',
-            "context": {'default_employee_id': self.id},
+            "context": context,
+            "name": "Time Off Requests",
+            "target": "new",
         }
```

Comment:
IMO this should still redirect towards a single leave view if there is only one selected employee, just for the sake of ensuring good computation of duration etc. In special cases related to l10ns :thinking: 
So something like this:
```py
    def action_open_leave_request(self):
        if len(self) == 1:
            model = 'hr.leave'
            context = {'default_employee_id': self.id}
        else:
            model = 'hr.leave.generate.multi.wizard'
            context = {
                'default_employee_ids': self.ids,
                'default_date_from': fields.Date.today(),
                'default_date_to': fields.Date.today(),
                'default_name': _('Unplanned Absence'),
            }

        return {
            'type': 'ir.actions.act_window',
            'res_model': model,
            'views': [[False, 'form']],
            'view_mode': 'form',
            'context': context,
            'target': 'new',
        }
```

Reply:
I have done the changes.

Thanks :slightly_smiling_face: